nodejs/week4/spam-detector/index.js

```diff
@@ -0,0 +1,89 @@
+class Email {
+	constructor(subject, body) {
+		this.subject = subject;
+		this.body = body;
+	}
+}
+
+class SpamDetector {
+	badWordCheck(email) {
+		const badWords = ["viagra", "offer", "free", "business proposal"];
+
+		const emailBody = email.body;
+		const emailBodyIntoArray = emailBody.split(" ");
+
+		const badWordsInEmail = emailBodyIntoArray.filter((item) => {
+			return badWords.includes(item.toLowerCase());
+		});
+
+		if (badWordsInEmail.length !== 0) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	subjectCheck(email) {
+		const emailSubject = email.subject;
+
+		if (emailSubject.toLowerCase() === "hello") {
+			return true;
+		} else {
+			return false;
+		}
+	}
```

Comment:
`emailSubject` or `email.subject` is about the same, so there is no need to restructure

```suggestion
		return email.subject.toLowerCase() === "hello"
	}
```

Reply:
okay, made the correction.